cli/zone.go

```diff
@@ -93,7 +92,7 @@ func runGetZone(cmd *cobra.Command, args []string) {
 		log.Errorf("Object %d: no zone config found", id)
 		return
 	}
-	fmt.Fprintln(osStdout, rows[0][1])
+	fmt.Println(rows[0][1])
```

Comment:
it would just be simpler to have the tests overwrite os.Stdout like everything else. having a custom variable in real code is a bit weird.